web/source/kmwembedded.ts

```diff
@@ -554,34 +561,35 @@ namespace com.keyman.osk {
    *  @param  {string}  keyName
    *  @return {boolean}         true if key code successfully processed
    */
-  keymanweb.processDefaultMapping = function(code, shift, Lelem, keyName) {
+  keymanweb.processDefaultMapping = function(keyEvent: com.keyman.text.KeyEvent, target: com.keyman.text.OutputTarget) {
     // Note:  this assumes Lelem is properly attached and has an element interface.
     // Currently true in the Android and iOS apps.
     let Codes = com.keyman.text.Codes;
+    let code = keyEvent.Lcode;
 
     // Default handling for external keys.
     // Intentionally not assigning K_TAB or K_ENTER so KMW will pass them back
     // to the mobile apps to handle (insert characters or navigate forms).
     if (code == Codes.keyCodes.K_SPACE) {
-      kbdInterface.output(0, Lelem, ' ');
+      kbdInterface.output(0, target, ' ');
       return true;
     } else if (code == Codes.keyCodes.K_BKSP) {
       kbdInterface.defaultBackspace();
```

Comment:
Does `defaultBackspace()` need to be changed too? In the KMW test page, the external keyboard "backspace" doesn't work.

Reply:
the backspace issue was only on Linux. It works on Windows

Reply:
Interesting.  Which browser(s) on Linux?  Might it be related to / a conflict with Keyman for Linux?  (I sometimes have 'interesting' interactions with Keyman Desktop on Windows when testing Web.)

Reply:
It was Chrome. I don't think I had Keyman for Linux installed at that point, but I'll check next time I'm back in office.

Reply:
Confirmed on another Linux setup (no Keyman for Linux installed).
* external backspace key didn't work.
* special tricolor key on `sil_cameroon_qwerty` doesn't work on the default layer

Same setup with `stable-11.0`
* external backspace key works
* tricolor key on `sil_cameroon_qwerty` works on the default layer

Reply:
Looks like I can get the same results in macOS Chrome.  So, what's Windows doing differently here...?

Reply:
Good. Your latest commits fixed all the above issues